src/bootstrap/rm.js

```diff
@@ -1,32 +1,28 @@
 'use strict'
 
-const promisify = require('promisify-es6')
+const Multiaddr = require('multiaddr')
+const configure = require('../lib/configure')
 
-module.exports = (send) => {
-  return promisify((args, opts, callback) => {
-    if (typeof opts === 'function' &&
-      !callback) {
-      callback = opts
-      opts = {}
+module.exports = configure(({ ky }) => {
+  return async (addr, options) => {
+    if (addr && typeof addr === 'object' && !Multiaddr.isMultiaddr(addr)) {
+      options = addr
+      addr = null
     }
 
-    // opts is the real callback --
-    // 'callback' is being injected by promisify
-    if (typeof opts === 'function' &&
-      typeof callback === 'function') {
-      callback = opts
-      opts = {}
-    }
+    options = options || {}
 
-    if (args && typeof args === 'object') {
-      opts = args
-      args = undefined
-    }
+    const searchParams = new URLSearchParams(options.searchParams)
+    if (addr) searchParams.set('arg', `${addr}`)
+    if (options.all != null) searchParams.set('all', options.all)
+
+    const res = await ky.post('bootstrap/rm', {
```

Comment:
Looking at the commit history, what happened with `DELETE` here?  Our HTTP API is all `POST`, all the time so this is fine like this, but that aside I thought `DELETE` was [ok to use in browsers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE#Browser_compatibility)?

Reply:
I couldn't be bothered to dig in at the time but thinking about it now it's because browsers are respecting CORS and our default `ipfsd-ctl` config does not include DELETE https://github.com/ipfs/js-ipfsd-ctl/blob/7928c1eee092a9b96a59328294c84c9f065217a3/src/defaults/config.json#L5-L9

Reply:
Do you want me to add DELETE ?
In this case i would use POST cause its more RPC than REST
i would only use DELETE if the url was https://host/bootstrap/{addr}

Reply:
I'm not suggesting changing this to `DELETE`, I just wanted to know what happened 😉